### DISCUSSION TITLE
Question about GT

### DISCUSSION THREAD

**Discussion by tiruss** [fredzzhang/hicodet#30] [Q&A] [answered]
What a great job!

I am deeply impressed with your work.

I have one question about GT("instance_{train2015, test2015}.json).

In the bounding box annotation("boxes_h" and "boxes_o"), there is slightly difference on same human and object like             "boxes_h": [
                [
                    208.0,
                    33.0,
                    427.0,
                    300.0
                ],
                [
                    213.0,
                    20.0,
                    438.0,
                    357.0
                ],
                [
                    206.0,
                    33.0,
                    427.0,
                    306.0
                ],
                [
                    209.0,
                    26.0,
                    444.0,
                    317.0
                ]

Is this because it came from the detector, not the GT conversion?

**Comment by fredzzhang** [comment, marked as answer]
Hi @tiruss,

The HICO-DET dataset was collected by [Chao et al.](http://www-personal.umich.edu/~ywchao/hico/), I merely implemented some utilities. The annotation files are in fact converted from the original files.

I don't know why the boxes are different for the same object instance, but my guess is that the annotations were collected class by class. As a result, the same bounding box for a different interaction class might be annotated by a different person, hence the minor difference.

Fred.
